integration-tests/contracts/ethereum_vrf_contracts.go

```diff
@@ -705,26 +705,37 @@ func (dkgContract *EthereumDKG) WaitForTransmittedEvent() (*dkg.DKGTransmitted,
 		return nil, err
 	}
 	defer subscription.Unsubscribe()
-	transmittedEvent := <-transmittedEventsChannel
-	if err != nil {
-		return nil, err
+
+	for {
+		select {
+		case err = <-subscription.Err():
+			return nil, err
+		case <-time.After(time.Minute * 5): // this can take a while
```

Comment:
can we provide the timeout from the test itself? rationale - to have just right amount of information available from test itself; for the test to be more descriptive